### ISSUE TITLE
Bad File Detected upload all media files

### ISSUE THREAD

**Issue by tyupika93** [e107inc/e107#3555]
I would like to download an image for the download, but for each image it is written by Bad File Detected. The format has no problem

**Comment by Moc** [comment]
Are you using the latest files from Github?

**Comment by tyupika93** [comment]
2.1.9 is maintained. I'm not updating the entire release of Github

**Comment by Jimmi08** [comment]
Today latest github, it looks it works now.

**Comment by Moc** [comment]
@tyupika93 I am closing this issue as I believe the issue is resolved in the latest files, which will be part of the 2.2.0 release. 

Please let me know if the issue persists, then I'll re-open it.

**Comment by Jimmi08** [comment]
@Moc There should be some explanation what is considered as bad file now.  I couldn't get it from code.  
The same steps - I make printscreen, saved as jpg or png (with Snagit) -  from (of) ten pictures one is detected as wrong and I can't it upload. No idea why.

**Comment by Moc** [comment]
@Jimmi08 Agreed. Please post it up as a new issue and I'll label it as enhancement.
